ocaml/idl/datamodel.ml

```diff
@@ -150,6 +150,10 @@ let _pgpu = "PGPU"
 let _gpu_group = "GPU_group"
 let _vgpu = "VGPU"
 let _vgpu_type = "VGPU_type"
+let _pvs_farm = "PVS_Farm"
```

Comment:
I think that you need to put you new class under `all_system` at the end of the file.